packages/documentation-site/patternfly-docs/pages/home.css

```
@@ -88,62 +41,25 @@
   }
 }
 
-.pf4-c-background-lines {
-  width: 100%;
-  max-width: 100%;
-  background-image: url("./lines-background.svg");
-  background-repeat: no-repeat, no-repeat;
-  background-position: 100% 0,100%;
+.pf-v5-theme-dark .pf-v5-l-flex.pf-m-flex-3-on-md.pf-v5-c-image__light-showcase {
+  display: none;
 }
 
-.pf4-c-image__laptop {
-  height: 40%;
-  width: 100%;
-  background-repeat: no-repeat;
-  background-position: 50%;
-  background-image: url("../images/laptop.svg");
+.pf-v5-l-flex.pf-m-flex-3-on-md.pf-v5-c-image__dark-showcase {
+  display: none;
 }
 
-.pf4-c-image__phone {
-  position: relative;
-  top: -120px;
-  height: 25%;
-  width: 100%;
-  background-repeat: no-repeat;
-  background-position: 32%;
-  background-image: url("../images/phone.svg");
+.pf-v5-theme-dark .pf-v5-l-flex.pf-m-flex-3-on-md.pf-v5-c-image__dark-showcase {
+  display: block;
 }
 
-.pf4-c-image__screen {
-  position: relative;
-  top: -225px;
-  height: 25%;
-  width: 100%;
-  background-repeat: no-repeat;
-  background-position: 60%;
-  z-index: 2;
-  background-image: url("../images/screen.svg");
-}
 
-.pf4-c-image__desktop {
-  position: relative;
-  top: -300px;
-  height: 42%;
-  width: 100%;
-  background-repeat: no-repeat;
-  background-position: 42%;
-  background-image: url("../images/desktop.svg");
+section.pf-v5-c-page__main-section.pf-m-limit-width.v5-featured-posts>div.pf-v5-c-page__main-body {
+  padding-bottom: 0px;
 }
 
-.pf4-c-background-image-principles {
-  background-position-x: 100%, 100%;
-  background-position-y: 0, 50%;
-  background-repeat: no-repeat;
-  background-image: url("./principles-background.svg"), linear-gradient(180deg,#06c,#004080);
-}
-
-.ws-homepage-main-section {
-  padding: var(--pf-v5-global--spacer--xl) var(--pf-v5-global--spacer--2xl) var(--pf-v5-global--spacer--2xl);
+section.pf-v5-c-page__main-section.pf-m-limit-width.ws-homepage-main-section>div.pf-v5-c-page__main-body {
```

As long as `ws-homepage-main-section` is only used on this page section, you can rewrite this as

```suggestion
.ws-homepage-main-section>.pf-v5-c-page__main-body {
```

You could also rewrite this block as


```
.ws-homepage-main-section {
    --pf-v5-c-page__main-section--PaddingRight: 0;
}
```